### PR TITLE
Add github-release binary

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add `github-release` 

## why
* so we can easily attach binaries to releases from codefresh in the build-harness

### CODE REVIEW

File: install/Makefile [hunk 1]
```diff
@@ -48,6 +48,15 @@ export GITHUB_COMMENTER_VERSION ?= 0.1.0
 github-commenter:
 	$(call github_download_binary_release,cloudposse,$(GITHUB_COMMENTER_VERSION),$@_$(OS)_$(ARCH))
 
+export GITHUB_RELEASE_VERSION ?= 0.7.2
+## Install github-release to create and edit releases on Github (and upload artifacts)
+github-release:
+	mkdir -p $(TMP)/$@
+	$(CURL) -o - https://github.com/aktau/$@/releases/download/v$(GITHUB_RELEASE_VERSION)/$(OS)-$(ARCH)-$@.tar.bz2 | tar -C $(TMP)/$@ -zx bin/$(OS)/$(ARCH)/$@
```
Comment (sarkis) [inline install/Makefile]: Should we use `-sSL` flag here with curl?

-s - silent
-S - show failure (when used with -s)
-L - follow symlink - unlikely chance that github forwards this some day (i.e. change of username?)

Comment (osterman) [reply install/Makefile]: Yea, good catch.

Comment (osterman) [reply install/Makefile]: Actually, we do this already: 

https://github.com/cloudposse/packages/blob/master/install/Makefile#L4

Comment (osterman) [reply install/Makefile]: (was surprised how it could have worked without it)

Comment (sarkis) [reply install/Makefile]: Oh good! I looked at the wrong `CURL` before commenting... which didn't have the flags (I think in build-harness) - thanks for linking that...

**Pull request merged**